### ISSUE TITLE
Issues with switching Audio device on Android

### ISSUE THREAD

**Issue by imasheep1** [doukutsu-rs/doukutsu-rs#189]
The game audio will stop working if the audio device is switched (For example, turning off Bluetooth earbuds). All audio will be muted until the app restarts regardless of the device volume

Tested with connecting and disconnecting both Bluetooth and Wired headphones, but the result is the same

**Comment by jozsefsallai** [comment]
I remember fixing this on the SDL backend but I don't think I ported the change over to the glutin backend.

**Comment by alula** [comment]
The audio subsystem is separate from rendering/windowing. Probably cpal being a meme again.

**Comment by jozsefsallai** [comment]
You're right, the changes I made should be independent from the rendering backend so it's weird that it does that. It's still not a perfect fix, as music playback won't resume until a new song is loaded (i.e. through `<CMU`).
